oopsla2024-benchmarks/Cargo.toml

```diff
@@ -0,0 +1,25 @@
+[package]
+name = "cedar-benchmarks"
+version = "0.1.0"
+edition = "2021"
```

Comment:
Why 2021 and not 2024?

Reply:
I think it's auto generated by cargo. The edition number is not equal to the current year.

Reply:
2021 is the latest edition as of this writing.  Note that crates of any edition are compatible with dependencies of both earlier and later editions; the decision of which edition to use is independent and local to each crate.  See https://doc.rust-lang.org/edition-guide/editions/.